system/gatekeeper/templates/constrainttemplate-region-value-mismatch.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: templates.gatekeeper.sh/v1
+kind: ConstraintTemplate
+metadata:
+  name: gkregionvaluemismatch
+spec:
+  crd:
+    spec:
+      names:
+        kind: GkRegionValueMismatch
+      validation:
+        # Schema for the `parameters` field
+        openAPIV3Schema:
+          type: object
+          properties:
+            helmManifestParserURL:
+              type: string
+            region:
+              type: string
+
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      libs:
+        - |
+          {{ .Files.Get "lib/add-support-labels.rego" | nindent 10 }}
+        - |
+          {{ .Files.Get "lib/helm-release.rego" | nindent 10 }}
+      rego: |
+        package gkregionvaluemissing
+        import data.lib.add_support_labels
+        import data.lib.helm_release
+
+        iro := input.review.object
+        release := helm_release.parse_k8s_object(iro, input.parameters.helmManifestParserURL)
+        violation[{"msg": release.error}] {
+          release.error != ""
+        }
+
+        violation[{"msg": add_support_labels.from_k8s_object(iro, msg)}] {
```

Comment:
You need to examine the owner-info. `iro` is a Helm release which does not have the right labels.